pagopa/io-dev-api-server
# chore: [IOBP-667,IOBP-666] Adapt user wallet definitions with the new openapi

## Short description
This PR adapts the new user wallet definition upgrading the openapi file

## List of changes proposed in this pull request
- Adapted details of a method with type = `PAYPAL` adding the new pspBusinessName metadata
- Adapted the `clients` attribute with the `IO` key;

## How to test
Try this PR combined with this other PR on the io-app: https://github.com/pagopa/io-app/pull/5803 , you should be able to see a pre-selected payment method and if it is a "Pagamento veloce con PayPal", you should be able to see also the email and PSP name.
